src/components/menu/Connect.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+import Panel from './Panel';
+
+export default function Connect() {
+  const [gameId, setGameId] = useState('');
+
+  return (
+    <>
```

Comment:
to trzeba zamieszczac w ten fragment? co jak to wyjebiesz

Reply:
Chodzi Ci o to <></>? Nie musi byc konkretnie to, może być np. <div> albo <React.Fragment>, <> to wersja skrócona. Musi być opakowane w jakiś markup.

Reply:
wiem ze moze byc to albo to, pytam czy to w ogole potrzebne, co sie stanie jak to wyjebiesz calkowicie i odrazu bedzie ten Panel

Reply:
Dobra, wygląda na to, że działa. Traversy gadał w tym filmiku, że musi być takie opakowanie, ale chyba jak nie ma innego opakowania w postaci komponentu innego albo jakiegoś diva np.